### PR TITLE
Pay advances pay checks payroll

### LINKED ISSUE #2788: Pay advances and paychecks with the Payroll module
When you automatically deduct the advance or paycheck automatically with the Payroll module, it is necessary to add the identifier of the creditor_uuid of the employee for this it is necessary to add a property in the Table Rubric at the end of allowing the adjournment of the employee's creditor_uuid when deducting the advance and the payday deposit

### PR DESCRIPTION
**Pull request by lomamech**
refactor(PayAdvance and Paycheck)
- Add property is_associated_employee for track to track the rubrics for
  which the employee's creditor_uuid must be added

refactor(Payroll Rubric)  …
- Update form and Rubric Registry, add property is_associated_employee to
  determine the topics that will be associated with the employee ID

bug(Payroll Multiple Commitment)  …
- removing of the discount headings in the list of taxable headings, when
  configuring the calculation of the payroll

Closes #2788

### CODE REVIEW

File: client/src/modules/payroll/rubrics/templates/associated.tmpl.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="ui-grid-cell-contents text-left">
```
Comment (jniles) [inline client/src/modules/payroll/rubrics/templates/associated.tmpl.html]: Do you need `text-left` here?

**Review by jniles — APPROVED** [review]
LGTM.

bors r+

**Comment by jniles** [comment]
:/  Looks like there are some things to fix here.

**Comment by lomamech** [comment]
Now All Bugs are fixed

**Comment by jniles** [comment]
Great!  bors r+

**Pull request merged**